script/common/check_xcode_version

```diff
@@ -2,11 +2,19 @@ check_xcode_version ()
 {
   if [ -z "$DISABLE_XCODE_CHECK" ]
   then
-    installed_xcode_version=`xcodebuild -version | head -n 1 | ruby -e 'puts STDIN.read.match(/Xcode ((\d+\.)?\d+\.\d+)/)[1]'`
-    if [ "$installed_xcode_version" != "$REQUIRED_XCODE_VERSION" ]
-    then
-      printf "\033[1;31mError: xcodebuild version '$installed_xcode_version' is not equal to '$REQUIRED_XCODE_VERSION'"
-      printf "\033[0m"
+    installed_xcode_version = `xcodebuild -version`
```

Comment:
I forgot to mention but the new script is written in Ruby. So that `if` statement won't work. This is how the file should look like

```ruby
#!/usr/bin/env ruby

return if ENV['DISABLE_XCODE_CHECK']

installed_xcode_version = `xcodebuild -version`
  .split("\n")
  .first
  .match(/Xcode ((\d+\.)?\d+\.\d+)/)[1]
required_xcode_version = ARGV.first

# Drop patch number
installed_xcode_version = installed_xcode_version[0..-3] if installed_xcode_version.scan(/\./).length == 2
required_xcode_version = required_xcode_version[0..-3] if required_xcode_version.scan(/\./).length == 2

if installed_xcode_version != required_xcode_version
  printf "\033[1;31mError: xcodebuild version '#{installed_xcode_version}' is not equal to '#{required_xcode_version}'"
  printf "\033[0m\n"
  exit 1
end
```